### PR TITLE
Removing unused code and renaming structures

### PR DESCRIPTION
**Pull request by lauravuo-techlab**
Preparing for routing keys handling and doing some code cleanup at the same time

### CODE REVIEW

File: std/didexchange/models.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ type Response struct {
 	ConnectionSignature *ConnectionSignature `json:"connection~sig,omitempty"`
 	Thread              *decorator.Thread    `json:"~thread,omitempty"`
 
-	connection *Connection // Actual data, to be signed or verified
+	Connection *Connection `json:"-"` // Actual data, to be signed or verified
```
Comment (techlab-lainio) [inline std/didexchange/models.go:31]: I could not see here why this need to be exported, but I suppose there is a reason?

File: agent/pairwise/pairwise.go [hunk 2]
```diff
@@ -11,14 +11,6 @@ import (
 	"github.com/lainio/err2"
 )
 
-type Saver interface {
```
Comment (techlab-lainio) [inline agent/pairwise/pairwise.go:14]: Cool!!

**Review by techlab-lainio — APPROVED** [review]
Nice, one question there was. Good job.

File: std/didexchange/models.go (hunk 1 above)
Comment (lauravuo-techlab) [reply std/didexchange/models.go:31]: It's needed in the coming PR 🙂 Dropped it out from json output so that there are no effect on protocol messages.

**Pull request merged**